src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -10,92 +9,234 @@ import { NewSearchBarInput_viewer$data } from "__generated__/NewSearchBarInput_v
 
 const logger = createLogger("Components/Search/NewSearchBar")
+const SEARCH_DEBOUNCE_DELAY = 250
 
 export interface NewSearchBarInputProps extends SystemContextProps {
   relay: RelayRefetchProp
   viewer: NewSearchBarInput_viewer$data
   isXs: boolean
 }
 
+const reportPerformanceMeasurement = performanceStart => {
+  const duration = performance.now() - performanceStart
+  const deviceType = getENV("IS_MOBILE") ? "mobile" : "desktop"
+
+  const metricPayload = {
+    name: "autocomplete-search-response",
+    tags: [`device-type:${deviceType}`, "design:rich"],
+    timing: duration,
+    type: "timing",
+  }
+
+  request
+    .post(getENV("VOLLEY_ENDPOINT"))
+    .send({
+      metrics: [metricPayload],
+      serviceName: "force",
+    })
+    .end()
+}
+
+const shouldStartSearching = (value: string) => {
+  return value.length > 1
+}
+
 const NewSearchBarInput: FC<NewSearchBarInputProps> = ({
   isXs,
   relay,
   viewer,
 }) => {
+  const tracking = useTracking()
   const { t } = useTranslation()
-  const [value, setValue] = useState("")
+  const [value, setValue] = useState(getSearchTerm(window.location))
+  const [selectedPill, setSelectedPill] = useState<PillType>(TOP_PILL)
+  // We use fetchCounter together with useUpdateEffect to track typing
+  const [fetchCounter, setFetchCounter] = useState(0)
+  const { router } = useRouter()
+  const encodedSearchURL = `/search?term=${encodeURIComponent(value)}`
 
   const options = extractNodes(viewer.searchConnection)
-  const formattedOptions = options.map(option => {
-    return {
-      text: option.displayLabel!,
-      value: option.displayLabel!,
-      subtitle: getLabel({
-        displayType: option.displayType ?? "",
+  const formattedOptions: SuggionItemOptionProps[] = options.map(
+    (option, index) => {
+      return {
+        text: option.displayLabel!,
+        value: option.displayLabel!,
+        subtitle:
+          getLabel({
+            displayType: option.displayType ?? "",
+            typename: option.__typename,
+          }) ?? "",
+        imageUrl: option.imageUrl!,
+        showArtworksButton: !!option.statuses?.artworks,
+        showAuctionResultsButton: !!option.statuses?.auctionLots,
+        href: option.href!,
         typename: option.__typename,
-      }),
-      imageUrl: option.imageUrl!,
-      showArtworksButton: !!option.statuses?.artworks,
-      showAuctionResultsButton: !!option.statuses?.auctionLots,
-      href: option.href!,
+        item_number: index,
+        item_type: option.displayType!,
+      }
     }
-  })
+  )
 
-  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    setValue(event.target.value)
+  // Clear the search term once you navigate away from search results
+  useMemo(() => {
+    router.addNavigationListener(location => {
+      if (!location.pathname.startsWith("/search")) {
+        setValue("")
+      }
+
+      return true
+    })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why disable? If ever we do this we need a clear comment describing the reason why.